src/chat/src/Bridge/Cloudflare/MessageStore.php

```diff
@@ -43,23 +43,38 @@ public function __construct(
     ) {
     }
 
+    /**
+     * @param array{
+     *     page?: int,
```

Comment:
I don't think `page` should be exposed as a public option. You should rather use a private method to load the appropriate page of the KV namespaces.

This private method could even be common with the logic of `retrieveCurrentNamespaceUuid` to get the current namespace Uuid (the *only* part of the logic you don't want is the exception when there is no namespace).